docs/guides/admin/docs/upgrade.md

```diff
@@ -82,11 +83,61 @@ change `maintenance` back to `false` to put the scheduler back into its normal m
 You should avoid running Opencast 7 without migrating the scheduled events first. Otherwise, your capture agents may
 fetch an empty calendar.
 
+
+Update Player Links
+-------------------
+
+> This step is optional
+
+Opencast 7 comes with the capability of dynamically switching the configured player without requiring the republication
+of all published material by providing the dynamic target `https://example.opencast.org/watch/<id>`. This new target
+will be used for all new engage publications.
+
+But old publications still reference the players directly in the admin interface and the external API. This does not
+pose any immediate problem unless you actually want to switch players in which case you would need to republish the
+material once to update the publication links.
+
+Alternatively, you can rewrite the old links all at once without re-publication using the following method:
+
+1. Find the archive directory and run the following command (replacing `<playerlink>`):
+
+        sed -i 's_<playerlink>_/watch/_g' .../archive/*/*/*/manifest.xml
+
+    For Theodul, Paella and the Engage player the specific commands would be:
+
+        sed -i 's_/engage/theodul/ui/core.html?id=_/watch/_g' .../archive/*/*/*/manifest.xml
+        sed -i 's_/paella/ui/watch.html?id=_/watch/_g' .../archive/*/*/*/manifest.xml
+        sed -i 's_/engage/ui/watch.html?id=123_/watch/_g' .../archive/*/*/*/manifest.xml
```

Comment:
Is `123` some kind of magic Engage ID prefix? (No sarcasm, I genuinely don't know.) :sweat_smile:

Reply:
No, it's not. It's my test id. Removed.